### ISSUE TITLE
playercreateinfo_* do not use a bitmask for classes/races

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/azerothcore-wotlk#3113]
##### SMALL DESCRIPTION:

In many parts of the core or the DB, we can combine values of a bitmask to target multiple classes/races. It's very practical, it saves time (and space xD). But in playercreateinfo_* tables, it's not the case, we have to put 1 row for each class, 1 for each race etc, just imagine the number of possibilities, it's shit :(

##### EXPECTED BLIZZLIKE BEHAVIOUR:

It should work like that: 3 = paladin + warrior

https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2129967/ChrClasses
https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2129966/ChrRaces

##### CURRENT BEHAVIOUR:

Actually it's like that: 3 = hunter

##### STEPS TO REPRODUCE THE PROBLEM:

1. Just fill the values in playercreateinfo_item and you'll get it

##### EXTRA NOTES:

- Of course, you will have to change the datatypes in the DB and maybe in core in ObjectMgr.cpp (like instead of `int32 current_class = fields[1].GetUInt8();` maybe just change to `uint16 current_class = fields[1].GetUInt16();`). For the SQL query it's something like that
```sql
ALTER TABLE `playercreateinfo_item`
CHANGE `class` `class` smallint(6) unsigned NOT NULL AFTER `race`;
```
- Plz if someone fixes it, tell me on discord in PM :D

##### BRANCH(ES):

master

**Comment by BarbzYHOOL** [comment]
Added extra info

**Comment by Kitzunu** [comment]
I tried doing this, got 800+ new errors, and gave up hahaha

**Comment by Shard-MW** [comment]
No point to "fix" it, since every race/class should have a different value in `playercreateinfo`, and `playercreateinfo_action`. 
Why not in `playercreateinfo_item` but this table isn't really used...

PS: `playercreateinfo_spell` is using bitmasks :p

**Comment by BarbzYHOOL** [comment]
> No point to "fix" it, since every race/class should have a different value in `playercreateinfo`, and `playercreateinfo_action`.
> Why not in `playercreateinfo_item` but this table isn't really used...
> 
> PS: `playercreateinfo_spell` is using bitmasks :p

yes it's still useful if you are creative even in playercreateinfo_action, stop advocating for inconsistency and time wasting shit

**Comment by pangolp** [comment]
Hello, good afternoon. Reading the comments a little, it might be a good idea. But the problem is that I don't know what to do with the topic at the moment. Should I add the label that a developer is needed or should we keep it that way for the time being?

**Comment by FrancescoBorzi** [comment]
@pangolp I think this is still valid, so no action required from your side. Thanks for checking!
